Header header
float32 major_axis
float32 minor_axis
float32 orientation
float32 x
float32 y
